### DISCUSSION TITLE
individual parts?

### DISCUSSION THREAD

**Discussion by Sewshi** [LibreScore/dl-librescore#12] [Q&A] [answered]
what happened to downloading individual parts?

**Comment by sealsrock12** [comment, marked as answer]
To download individual parts, first download the MSCZ, then use the webmscore website to convert: https://webmscore-pwa.librescore.org/. You can download MSCZ with the LibreScore app: https://github.com/LibreScore/app-librescore#installation.

Individual parts always relied on the MSCZ file, and now that MSCZ is not available in the script anymore, neither are individual parts.
